inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -253,16 +266,43 @@ def plot(
               the matplotlib `Axes` object is returned instead.
         """
 
+        # Create uniform grid for interpolation.
+        x_grid, y_grid = np.meshgrid(np.linspace(*self.x_range, grid_size),
+                                     np.linspace(*self.y_range, grid_size))
+
+        # Interpolate depths to uniform grid.
+        interpolator = scipy.interpolate.LinearNDInterpolator(
+            (self.x, self.y),
+            self.depths,
+        )
+
+        depths_grid = interpolator(x_grid, y_grid)
+
+        # Filter out points that are far from bathymetry locations.
+        tree = scipy.spatial.KDTree(np.c_[self.x, self.y])
+        distance, _ = tree.query(np.c_[x_grid.ravel(), y_grid.ravel()], k=1)
```

Comment:
I feel that this is very weird. What distance are you obtaining here?

Reply:
I am obtaining the distance between all points defining the tree, (i.e. all points where the bathymetry is defined,`(self.x, self.y)`) and all points of the uniform grid, `(x_grid.ravel(), y_grid.ravel())`.